src/connectors/hits-per-page/connectHitsPerPage.ts

```diff
@@ -162,7 +149,7 @@ const connectHitsPerPage: HitsPerPageConnector = function connectHitsPerPage(
       setHitsPerPage: (value: HitsPerPageConnectorParamsItem['value']) => any;
       createURLFactory: (
         state: SearchParameters
-      ) => HitsPerPageRendererOptions['createURL'];
+      ) => HitsPerPageRendererOptions<{}>['createURL'];
```

Comment:
It'd make sense to forward the type, right?

```suggestion
      ) => HitsPerPageRendererOptions<THitsPerPageWidgetParams>['createURL'];
```